src/i_winmusic.c

```diff
@@ -503,6 +503,32 @@ static void ResetDevice(void)
     }
 }
 
+static boolean IsPartLevel(const byte *msg, int length)
```

Comment:
Add your explanation from the PR here as a comment so someone only reading the source can see it too.

Reply:
Fixed by https://github.com/chocolate-doom/chocolate-doom/pull/1595/commits/21621e30af43af24e6bd8175c804250f2c337246.